WBooks/Extensions/UIAlertControllerExtension.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UIAlertControllerExtension.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 30/04/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+
+extension UIAlertController {
+    func createErrorAlert(title: String = "", message: String = "", action: @escaping (UIAlertAction) -> Void = {_ in }) -> UIAlertController {
```

Comment:
default values could be nil

Reply:
i leave message  required without default and nil for the rest